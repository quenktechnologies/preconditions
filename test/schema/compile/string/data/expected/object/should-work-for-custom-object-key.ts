base.and(base.and(base.type("object"),object.schemaProperties({name : base.and(base.type("string"),stringify),id : base.type("string")}, object.restrict, base.and(base.type("number"),numberify))),objectify)