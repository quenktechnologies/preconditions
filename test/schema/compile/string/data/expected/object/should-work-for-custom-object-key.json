base.and(base.and(base.type("object"),base.or(object.restrict({name : base.and(base.type("string"),stringify),id : base.type("string")}),object.map(base.and(base.type("number"),numberify)))),objectify)